compiler/AST/build.cpp

```diff
@@ -1170,7 +1170,7 @@ static BlockStmt* buildLoweredCoforall(Expr* indices,
   if (bounded) {
     if (!onBlock) { block->insertAtHead(new CallExpr("chpl_resetTaskSpawn", numTasks)); }
     block->insertAtHead(new CallExpr("_upEndCount", coforallCount, countRunningTasks, numTasks));
-    block->insertAtHead(new CallExpr(PRIM_MOVE, numTasks, new CallExpr(".", iterator,  new_CStringSymbol("size"))));
+    block->insertAtHead(new CallExpr(PRIM_MOVE, numTasks, new CallExpr("chpl_coforallSize", iterator)));
```

Comment:
I think this could have been changed to use `sizeas(int)`. I don't think there's any need to preserve the index type here. I'm fine leaving it as-is, but I was confused why this change was needed at first.

Reply:
I think you may be right; I did it this way less to get `sizeAs(idxType)` and more because it seemed simpler to move the code to a module and edit it there over time rather than having to build up AST expressions in the compiler.  It was also one of the only .size queries in the compiler, so this had the benefit of getting them all into module code.